README.md

```diff
@@ -64,7 +64,7 @@ but not limited to:
 * CoAP
 * CCN-Lite
 * Sigfox
-* LoRaWAN (Experimental)
+* LoRaWAN
```

Comment:
Unrelated. Please split at least into a separate commit.

Reply:
+1 and I think we should still consider it as experimental.